src/modules/objects/KvsObject_pixmap.cpp

```diff
@@ -463,7 +463,20 @@ KVSO_CLASS_FUNCTION(pixmap, mirrored)
 		return false;
 	}
 	m_currentType = Image;
+
+#if (QT_VERSION < QT_VERSION_CHECK(6, 9, 0))
```

Comment:
As small performance optimization, perhaps skip this altogether, if both directions are false?